go.mod

```diff
@@ -39,6 +39,7 @@ require (
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.7.1
+	github.com/stretchr/testify v1.7.0
```

Comment:
This makes it very easy to assert errors. I noticed we were not using this package. Let me know if this is an overhead.

Reply:
While Testify is a very powerful tool, it is not a required library for this tool. For this reason, we have adopted a policy of not using Testify in this project.
I am sorry, but could you please replace it with the standard error package? 🙇

Reply:
**Updated the test to not use testify**. Also, thank you for raising this. I refactored the tests to use the table driven approach which makes them a lot easier to read, debug and reduces duplication. Thanks